curriculum/challenges/english/05-apis-and-microservices/apis-and-microservices-projects/exercise-tracker.md

```diff
@@ -46,7 +50,7 @@ tests:
       }
     }
     "
-  - text: I can get an array of all users by getting api/exercise/users with the same info as when creating a user.
+  - text: You can make a `GET` request to `api/exercise/users` to get an array of all users. Each element in the array is an object containing a user's `username` and `_id`.
```

Comment:
Just to mention it:
- Should we consider adding to the `testString` so that it matches the `text`/user-story more accurately?